sgp40_voc_index/sgp40_voc_index_example_usage.ino

```diff
@@ -59,14 +59,14 @@ void loop() {
   err = sensirion_measure_voc_index_with_rh_t(
           &voc_index, &relative_humidity_percent, &temperature_celsius );
   if (err == STATUS_OK) {
-    Serial.print("VOC Index: ");
-    Serial.println(voc_index);
-    Serial.print("Temperature: ");
-    Serial.print(temperature_celsius * 0.001f);
-    Serial.println("degC");
-    Serial.print("Relative Humidity: ");
-    Serial.print(relative_humidity_percent * 0.001f);
-    Serial.println("RH");
+    Serial.print("VOCindex:");
+    Serial.print(voc_index / 10.0f);
+    Serial.print("\t");
+    Serial.print("Humidity[%RH]:");
+    Serial.print(relative_humidity / 100.0f);
+    Serial.print("\t");
+    Serial.print("Temperature[degC]:");
+    Serial.println(temperature_celsius / 200.0f);
```

Comment:
This changes the factor from 0.001 to 0.005

Reply:
All the factors have actually changed here... Definitely an error, which means I'm a sloppy reviewer :wink:  Thanks!